core/logger/default.go

```diff
@@ -17,6 +18,12 @@ func init() {
 	if err != nil {
 		log.Fatalf("failed to register pretty printer %+v", err)
 	}
+	err = zap.RegisterSink("splunk", func(url *url.URL) (zap.Sink, error) {
```

Comment:
We don't want to register it at all if `LOG_TO_SPLUNK=false`

Reply:
OK.